.github/codeql/codeql-config.yml

```diff
@@ -80,7 +80,7 @@ paths-ignore:
   - x-pack/plugins/fleet/scripts
   - x-pack/plugins/lists/scripts
   - x-pack/plugins/lists/server/scripts
-  - x-pack/plugins/ml/scripts
+  - x-pack/plugins/shared/ml/scripts
```

Comment:
@walterra I was going to comment that this should be something like `x-pack/platform/plugins/shared/ml/scripts`, however, from what I can see, this path does not exist (nor did the former `x-pack/plugins/ml/scripts`). Maybe this should be removed?

Reply:
Great catch! Turns out we removed the directory in this PR: https://github.com/elastic/kibana/pull/190840 - I removed the line from this file in [43c37a7](https://github.com/elastic/kibana/pull/202764/commits/43c37a7044060d1eb7812d78f04dc28d96aa4452).